app.py

```
@@ -412,7 +468,65 @@ def chat():
+    Returns:
+        フォールバック処理の結果
+    """
+    try:
+        # 簡単なフォールバック応答
+        return {
+            "content": "申し訳ございません。一時的に問題が発生しています。しばらくしてから再度お試しください。",
+            "model": fallback_model,
+            "fallback": True
+        }
+    except Exception as e:
+        logger.error(f"フォールバック処理でエラー: {e}")
```

Ruff (0.12.2)

Undefined name `logger`

(F821)